### PR TITLE
docs: Update errorCodes ESM import and usage

### PR DESCRIPTION
**Pull request by melroy89**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: docs/Reference/Errors.md [hunk 1]
```diff
@@ -236,7 +236,8 @@ fastify.listen({ port: 3000 }, function (err, address) {
 You can access `errorCodes` for mapping:
 ```js
 // ESM
-import { errorCodes } from 'fastify'
+import fastify from 'fastify'
+fastify.errorCodes
```
Comment (Uzlopak) [inline docs/Reference/Errors.md:240]: Why?

**Review by Uzlopak — CHANGES_REQUESTED** [review]
This PR is not adding any value.

**Review by climba03003 — CHANGES_REQUESTED** [review]
Actually, you should move the `fastify.errorCodes = errorCodes` to `module.exports.errorCodes = errorCodes`.
Note that it must below `module.exports = fastify`.

File: docs/Reference/Errors.md (hunk 1 above)
Comment (climba03003) [reply docs/Reference/Errors.md:240]: Currently, it is not possible to do so.

**Comment by melroy89** [comment]
> This PR is not adding any value.

Well actually.. It's showing how ESM errorCodes are currently implemented in fastify. Hence the documentation request.

But we can also improve the code. Either way, the documentation is currently not reflecting how it now works.

**Comment by melroy89** [comment]
> Actually, you should move the `fastify.errorCodes = errorCodes` to `module.exports.errorCodes = errorCodes` in `fastify.js`. Note that it must below `module.exports = fastify`.

Like this: https://github.com/fastify/fastify/pull/5390 ?

**Pull request closed without merging**